server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -1015,15 +1014,13 @@ private Modal blockDeleteModal(
           .withId("block-delete-form")
           .with(
               div(
-                      h1(join(blockDefinition.name(), " includes ", b(listItemsInBlock + ".")))
-                          .withClasses("text-base", "mb-2"),
-                      h1("Are you sure you want to delete this screen?")
-                          .withClasses("text-base", "mb-2"))
-                  .withClasses("mx-4"),
+                  h1(join(blockDefinition.name(), " includes ", b(listItemsInBlock + ".")))
+                      .withClasses("text-base", "mb-2"),
+                  h1("Are you sure you want to delete this screen?")
+                      .withClasses("text-base", "mb-4")),
```

Comment:
So we are getting rid of mb-2 replacing it with mb-4? just checking if we still want to keep line 1018 as mb-2

Reply:
Yep, we want to keep mb-2 on the "includes" line. This is it:

<img width="902" alt="image" src="https://user-images.githubusercontent.com/30369272/228918502-a729fad3-9ff5-4096-9fab-b33f2805fd84.png">